scripts/lint.sh

```diff
@@ -94,4 +94,16 @@ if [ ! -z "$output" ]; then
 	exitcode=1
 fi
 
+# ShellCheck linting
+for file in `find . -name "*.sh"`
```

Comment:
For consistency reasons, how about using the `$(<bash command>)` syntax as used above, see line 68.

Reply:
As far as I know, backticks are considered "old style" and `$(..)` should be preferred.

Reply:
Thanks for pointing this out! :)